Switch deployment providers from Vercel to Netlify 🎉

EthanThatOneKid opened the pull request:
Resolves #697.

EthanThatOneKid committed:
Switch deployment providers from Vercel to Netlify pop

Resolves #697.

EthanThatOneKid committed:
Update documentation and .env.example

EthanThatOneKid committed:
Remove Vercel analytics

EthanThatOneKid committed:
Fix old link.

EthanThatOneKid committed:
Merge branch 'main' into fix/697

EthanThatOneKid committed:
Merge branch 'main' into fix/697

EthanThatOneKid committed:
npm uninstall `@vercel/analytics`

EthanThatOneKid committed:
Install esbuild dev dependency

EthanThatOneKid committed:
Install devalue

According to <https://github.com/sveltejs/kit/issues/6462#issuecomment-1244680791>

EthanThatOneKid committed:
Setup Netlify adapter

EthanThatOneKid committed:
Add netlify.toml

Thank you Rodney Lab: https://rodneylab.com/use-netlify-to-host-your-sveltekit-site/

EthanThatOneKid committed:
Update README.md

EthanThatOneKid committed:
Update README.md

charliettaylor commented on package.json:
```diff
@@ -29,16 +29,18 @@
-    "@sveltejs/adapter-auto": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/adapter-netlify": "^1.0.0-next.84",
+    "@sveltejs/kit": "^1.0.0-next.544",
```
Is there a specific reason these two are no longer `next`?

EthanThatOneKid committed:
Installed "next" versions of SvelteKit packages

Investigating <https://github.com/EthanThatOneKid/acmcsuf.com/pull/700#discussion_r1020969621>.

anhduy1202 reviewed (approved):
lgtm
